src/pages/Home/components/ContentRoutes/ContentRoutes.js

```diff
@@ -3,30 +3,37 @@ import { Route, Routes, Navigate } from "react-router-dom";
 import ProfileContent from "../ProfileContent/ProfileContent";
 import LoginSignupForm from "../LoginSignupForm/LoginSignupForm";
 import Editor from "../Editor/Editor";
+import ProtectedEditRoute from "../Editor/components/ProtectedEditRoute/ProtectedEditRoute";
+import { useContext } from "react";
+import { UserContext } from "containers/CognitoUserProvider/CognitoUserProvider";
```

Comment:
Doesn't look like `user` is being used anywhere in ContentRoutes, just being passed to ProtectedEditRoute. Why not just import this data in ProtectedEditRoute? 

Was it your intention to set this up to address issue #53 ?

Reply:
I think this was done during our troubleshooting prior to the commit. I remember you suggested it and I had proposed it originally, as a way to identify the user and have that date passed through the component. Feel free to remove it or we can keep it in place until #53 is addressed.

Reply:
just updated!